### DISCUSSION TITLE
How to make bookmarks clickable in fullscreen mode? Bookmark tab vanishes when mousing over.

### DISCUSSION THREAD

**Discussion by rgerg34g3g** [Floorp-Projects/Floorp#825] [Q&A]
https://github.com/Floorp-Projects/Floorp/assets/98705916/5dc8859c-0026-4fee-9b76-f4759d412f0d

**Comment by CutterKnife** [comment]
Sorry, I can't seem to reproduce this issue in my environment (macOS).
Instead, I recommend right-clicking on the toolbar and turning off the "Hide Toolbar" option.

**Reply by rgerg34g3g** [reply]
Did you try it in fullscreen mode? (press F11). What happens when you try it? Is it clickable? I have the bookmarks toolbar set to "always show".

**Reply by CutterKnife** [reply]
Yes, it is selectable on my Mac.
However, "Hide Toolbar" doesn't seem to work very well with macOS in the first place.
https://imgur.com/a/gLV0ZJd

**Comment by breadbeforerice** [comment]
I was able to reproduce this with a new profile in win10, but the bookmark toolbar did not appear in Firefox.
If you click once on the URL bar and then click on the tab bar, the bookmark toolbar will be fixed and you can press the bookmark.

**Reply by rgerg34g3g** [reply]
Yes, but that requires three clicks every time instead of just one. Ruins the browsing experience.
